downstream/modules/platform/ref-converting-playbook-examples.adoc

```diff
@@ -0,0 +1,189 @@
+----
+
+The volume mount is mapped with the same name in the container and has read-write capability. This will get used when you launch the job template.
+
+The _prompt on launch_ should be set for _extra_vars_ so you can adjust the sleep duration for each run, The default is 30 seconds.
+
+Once launched, and the _wait_for_ module is invoked for the sleep, you can go onto the execution node and look at what is running.   
+
+To verify the run has completed successfully, run this command to get an output of the job:
+
+----
+$ podman exec -it 'podman ps -q' /bin/bash
+bash-4.4#
+----
+
+You are now inside the running execution environment container.
+
+Look at the permissions, you will see that *awx* has become ‘root’, but this is not really root as in the superuser, as you are using rootless Podman, which maps users into a kernel namespace similar to a sandbox. Learn more about How does rootless Podman work? for shadow-utils.
+
+----
+bash-4.4# ls -la /mydata/
+Total 4
+drwxr-xr-x. 2 root root 41 Apr 28 09:27 .
+dr-xr-xr-x. 1 root root 77 Apr 28 09:40 ..
+-rw-r—--r–. 1 root root 33 Apr 11 12:34 file_read
+-rw-r—--r–. 1 root root  0 Apr 28 09:27 file_write
+----
+
+According to the results, this job failed. In order to understand why, the remaining output needs to be examined.
+
+
+----
+TASK [Read pre-existing file…]******************************* 10:50:12
+ok: [localhost] =>  {
+    “Msg”: “This is the file I am reading in.”
+
+TASK {Write to a new file...}********************************* 10:50:12
+An exception occurred during task execution. To see the full traceback, use -vvv. The error was: PermissionError: [Errno 13] Permission denied: b'/mydata/.ansible_tmpazyqyqdrfile_write' -> b' /mydata/file_write' 
+Fatal: [localhost]: FAILED! => {"changed": false, :checksum": "9f576o85d584287a3516ee8b3385cc6f69bf9ce", "msg": "Unable to make b'/root/.ansible/tmp/anisible-tim-1651139412.9808054-40-91081834383738/source' into /mydata/file_write, failed final rename from b'/mydata/.ansible_tmpazyqyqdrfile_write': [Errno 13] Permission denied: b'/mydata/.ansible_tmpazyqyqdrfile_write' -> b'/mydata/file_write}
+...ignoring
+
+TASK [Read written out file...] ****************************** 10:50:13
+Fatal: [localhost]: FAILED: => {"msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError;>, original message: could not locate file in lookup: /mydate/file_write. Vould not locate file in lookup: /mydate/file_write"}
+...ignoring
+----
+
+The job failed, even though *:rw* is set, so it should have write capability. 
+The process was able to read the existing file, but not write out. This is due to SELinux protection that requires proper labels to be placed on the volume content mounted into the container. If the label is missing, SELinux may prevent the process from running inside the container. Labels set by the OS are not changed by Podman. See the Podman documentation for more information.
```

Comment:
Do we need a cross-reference to the Podman docs here?

Reply:
Yes, and no. These are also referenced in the Reference module so as not to stunt the flow. The SME thought it was better to attempt to have the links elsewhere.